### DISCUSSION TITLE
SO behind Proxy/ loadbalancer.

### DISCUSSION THREAD

**Discussion by OzzyKampha** [Security-Onion-Solutions/securityonion#8889] [Unsupported Versions]
Hi. What type of field with FQDM does the ngnix need to get from the loadbalancer for getting past the cokie policy?

**Comment by reyesj2** [comment]
Not really sure I am understanding what the question is here. Can you expand on what you're trying to accomplish?

**Reply by OzzyKampha** [reply]
I have a Loadbalancer in front of SO. i have checked tcpdump locally+ checked the firewall logs locally. The HTTPS traffic hits SO and its not dropped in the firewall. But the browser is not showing anything, it just waiting until it is getting timeout.

While i try to visit SO direct without loadbalancer, i have no problem.

**Comment by dougburks** [comment]
Did you change `url_base` to the FQDN you use to connect to the load balancer?
https://docs.securityonion.net/en/2.3/url-base.html

**Reply by OzzyKampha** [reply]
Yes i did. Stil no response in the browser
